CMakeLists.txt

```diff
@@ -61,6 +61,16 @@ if (POLICY CMP0074)
 endif()
 project(OSRM C CXX)
 
+# add @loader_path/$ORIGIN to rpath to support relocatable binaries
+set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
```

Comment:
When I was testing this new package this check was red for me https://github.com/Project-OSRM/osrm-backend/blob/96f5780f06e6f55013762c879c361d647036bd72/scripts/ci/node_package.sh#L22

Tbh I don't know how it worked before, but it is how I fixed that.